cli/src/helper.rs

```diff
@@ -51,3 +55,41 @@ pub async fn bytes_from_hex_or_file(hex: Option<String>, file: Option<String>) -
         None
     })
 }
+
+pub async fn get_mnemonic() -> Result<String, Error> {
+    let mnemonic = generate_mnemonic()?;
+    println_log_info!("Mnemonic has been generated.");
+
+    let choices = [
+        "Write To Console Only",
+        "Write To File Only",
+        "Write To Console and File",
+    ];
+    let selected_choice = Select::with_theme(&ColorfulTheme::default())
+        .items(&choices)
+        .default(0)
+        .interact_on_opt(&Term::stderr())?;
+
+    if matches!(selected_choice, Some(0 | 2)) {
+        println!("{}", mnemonic);
+    }
+    if matches!(selected_choice, Some(1 | 2)) {
+        write_mnemonic_to_file(MNEMONIC_FILE_NAME, &mnemonic).await?;
+    }
+    Ok(mnemonic)
+}
+
+async fn write_mnemonic_to_file(path: &str, mnemonic: &str) -> Result<(), Error> {
+    let mut file = OpenOptions::new().create(true).append(true).open(path).await?;
+    file.write_all(format!("{mnemonic}\n").as_bytes()).await?;
+
+    println_log_info!("IMPORTANT:");
+    println_log_info!(
+        "Mnemonic has been written to '{MNEMONIC_FILE_NAME}'. Make sure it resides in a secure location."
+    );
+    println_log_info!(
+        "It is the only way to recover your account if you ever forget your password and/or lose the stronghold file."
+    );
```

Comment:
This warning should also be there if it's only written to console

Reply:
alright :+1: